### PR TITLE
Update graph-connector-agent.md

### PR DESCRIPTION
**Pull request by VivekKumarG**
Updating permissions requirement for GCA

### CODE REVIEW

File: MicrosoftSearch/graph-connector-agent.md [hunk 1]
```diff
@@ -63,7 +63,15 @@ First, sign-in and note that the minimum required privilege on the account is se
 
 6. Select **Microsoft Graph** and then **Application permissions**.
 
-7. Search for "ExternalItem.ReadWrite.All", "Directory.Read.All" and "ExternalConnection.ReadWrite.OwnedBy" from the permissions and select **Add permissions**.
+7. Search for the following permissions and select **Add permissions**.
+
+   | **Permission** | **When is the permission required** |
+   | ------------- | -------------|
+   | [ExternalItem.ReadWrite.OwnedBy](/graph/permissions-reference#application-permissions-52) | Always |
+   | [ExternalItem.ReadWrite.All](/graph/permissions-reference#application-permissions-52) | Optional |
```
Comment (jeffkizn) [inline MicrosoftSearch/graph-connector-agent.md:71]: @VivekKumarG THis permission is shown as optional but doesn't state why or when I should choose to add this. Can you clarify that point or delete this permission?

File: MicrosoftSearch/graph-connector-agent.md [hunk 2]
```diff
@@ -63,7 +63,15 @@ First, sign-in and note that the minimum required privilege on the account is se
...
+   | [Directory.Read.All](/graph/permissions-reference#application-permissions-23) | Required for File share, MS SQL and Oracle SQL connectors |
+  
+  <!--- | [ExternalConnection.ReadWrite.OwnedBy](/graph/permissions-reference#application-permissions-52) | Required for Graph connector SDK | --->
```
Comment (jeffkizn) [inline MicrosoftSearch/graph-connector-agent.md:74]: Why is this line included but commented? Can you include it or delete it?

**Review by jeffkizn — CHANGES_REQUESTED** [review]

File: MicrosoftSearch/graph-connector-agent.md (hunk 1 above)
Comment (VivekKumarG) [reply MicrosoftSearch/graph-connector-agent.md]: @jeffkizn Made the required change.

File: MicrosoftSearch/graph-connector-agent.md (hunk 2 above)
Comment (VivekKumarG) [reply MicrosoftSearch/graph-connector-agent.md]: @jeffkizn Deleted the comment

**Review by jeffkizn — APPROVED** [review]

**Pull request merged**